### PR TITLE
Datetime functions

### PR DESCRIPTION
**Pull request by bobvanratingen**

### CODE REVIEW

File: sqlglot/expressions.py [hunk 1]
```diff
@@ -1862,22 +1863,54 @@ class CurrentDatetime(Func):
...
+
 class DateDiff(Func, TimeUnit):
     arg_types = {"this": True, "expression": True, "unit": False}
 
 
+class DateTrunc(Func, TimeUnit):
+    arg_types = {"this": True, "unit": True, "zone": False}
+
+
+class DateTimeAdd(Func, TimeUnit):
```
Comment (tobymao) [inline sqlglot/expressions.py:1890]: should we change this to DatetimeAdd, notice the lower case t

File: sqlglot/expressions.py [hunk 2]
```diff
@@ -1862,22 +1863,54 @@ class CurrentDatetime(Func):
...
+    arg_types = {"this": True, "expression": True, "unit": False}
+
+
+class DateTimeSub(Func, TimeUnit):
```
Comment (tobymao) [inline sqlglot/expressions.py:1894]: DatetimeSub

**Comment by tobymao** [comment]
https://github.com/tobymao/sqlglot/commit/9b8bc67fd5f334077aa0031276adc186623f5c91

File: sqlglot/expressions.py (hunk 1 above)
Comment (bobvanratingen) [reply sqlglot/expressions.py:1890]: 👍

File: sqlglot/tokens.py [hunk 3]
```diff
@@ -104,8 +105,9 @@ class TokenType(AutoName):
     CUBE = auto()
     CURRENT_DATE = auto()
     CURRENT_DATETIME = auto()
-    CURRENT_ROW = auto()
+    CURRENT_TIME = auto()
     CURRENT_TIMESTAMP = auto()
+    CURRENT_ROW = auto()
```
[the remark this answers is not in the document]
Comment (bobvanratingen) [reply sqlglot/tokens.py:110]: Will do tomorrow !

[the remark this answers is not in the document]
Comment (tobymao) [reply sqlglot/tokens.py:110]: i did all the changes for you already

**Pull request merged**